device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/td3-a7050cx3-32s-32x100G.config.bcm

```diff
@@ -48,6 +48,11 @@ robust_hash_disable_vlan=1
 stable_size=0x5500000
 tdma_timeout_usec=15000000
 tslam_timeout_usec=15000000
+sai_optimized_mmu=1
+mmu_init_config="TD3-MSFT-T0-100G"
```

Comment:
Do we know if this particular SKU will only be used for T0 and NOT for T1 ever?
What if this SKU is also needed for T1 then this setting will not cause issue?

Reply:
Agreed. Working with BRCM on removing this from config.bcm to avoid such issues.